src/commands/tenmans.ts

```diff
@@ -226,3 +300,15 @@ const createQueueActionRow = (queueId) => {
       .setStyle(Constants.MessageButtonStyles.DANGER)
   );
 };
+
+export async function handleVoteCleaning() {
```

Comment:
Did you explore the advantages / disadvantages of just running `setTimeout` when a vote is created for the length of time of the vote? That way you'd get a more exact time that the vote turns off, and I should think it would make for easier handling

Reply:
I used setInterval because I planned on persisting queue state in the database for error tolerance, and setTimeout on vote creation will not work if the server crashes, whereas setInterval will allow us to regularly interact with the database to check queue state.